new_mvc/controllers/authController.js

```diff
@@ -41,47 +41,49 @@ exports.validateUser = async (req, res) => {
 
         const [row] = await auth.validateUser(req.body.login);
 
+        const dbUser = row[0];     //  !!!!!
```

Comment:
try to assign dbUser already, like `const dbUser = await auth.validateUser(req.body.login)?.row[0]`